pkg/apis/core/v1/defaults_test.go

```diff
@@ -682,73 +682,6 @@ func TestSetDefaultReplicationController(t *testing.T) {
 	}
 }
 
-func TestSetDefaultReplicationControllerReplicas(t *testing.T) {
```

Comment:
I knew people was working on this but I didn't know we were so close of removing the validation code ... it is amazing, is this for this release?

Reply:
Definitely not dropping go validation code this release yet… plenty of work to prove out the correctness and performance of the declarative stuff for builtins yet